### ISSUE TITLE
Screen Reader: Settings-Profile: The icon of new entry (red dot) is not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76045]
## Action Performed
Prerequisites:
- The user is signed in
- the user has new entries for Subscriptions or Inbox

1. Using JAWS + Chrome, open the web page https://new.expensify.com/settings/profile
2. Press Tab key to focus the buttons of Subscriptions and Inbox.
3. Note how the controls are displayed and announced.

## Expected Result
When users navigate to the 'Subscriptions' or 'Inbox' button, a visual indication of unread notifications will be displayed to the user; the same should be announced, e.g.:
- 'Subscriptions, Unread entries'
- 'Inbox, Unread entries'
or similar

## Actual Result
When users navigate to the account profile settings page and there is a new entry to view (indicated by a red dot next to 'Inbox' or 'Subscriptions') only the name of the button is announced; users have no way of knowing other than a visual indication that there are unread entries available.

HTML:
<div aria-label="Subscription" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-18u37iz r-1wtj0ep r-13qz1uu r-1awozwy r-1xfd6ze r-102gzdx r-11f147o r-3pj75a" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci r-iyfy8q"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M1.5 3C.7 3 0 3.7 0 4.5v1c0 .3.2.5.5.5h19c.3 0 .5-.2.5-.5v-1c0-.8-.7-1.5-1.5-1.5zm-1 5c-.3 0-.5.2-.5.5v7c0 .8.7 1.5 1.5 1.5h17c.8 0 1.5-.7 1.5-1.5v-7c0-.3-.2-.5-.5-.5z"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;">Subscription</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"><div class="css-g5y9jx r-1awozwy r-1777fci r-13hce6t"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#F25730"><circle cx="10" cy="10" r="7.5"></circle></svg></div></div></div></div></div></div>

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Update the current aria-label to include also the availability of new entries, e.g.:
- 'Subscriptions, Unread entries'
- 'Inbox, Unread entries'
or similar
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment:
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/3d9220ec-d98e-4b37-bccf-aa6f39f7f80b

<img width="1917" height="1003" alt="Image" src="https://github.com/user-attachments/assets/23698bb5-764e-4585-9f8b-ec085da1bd9f" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6986017
**Issue reported by:** NA
**Slack conversation:** NA
